crates/context_aware_config/src/api/dimension/handlers.rs

```diff
@@ -50,76 +52,175 @@ async fn create(
     let create_req = req.into_inner();
     let schema_value = create_req.schema;
 
-    validate_jsonschema(&state.meta_schema, &schema_value)?;
-
-    let schema_compile_result = JSONSchema::options()
-        .with_draft(Draft::Draft7)
-        .compile(&schema_value);
-
-    if let Err(e) = schema_compile_result {
-        return Err(bad_argument!(
-            "Invalid JSON schema (failed to compile): {:?}",
-            e
-        ));
-    };
+    let num_rows = dimensions
+        .select(count_star())
```

Comment:
nab: you get all columns if you remove this right? This is redundant I think.

Reply:
@Datron here I want the total number of rows , that's why count_star()

Reply:
@pratikmishra356 then you can call count() function on it